From: Jiri Slaby <jslaby@suse.cz>
Subject: kABI: reexport tcp_send_ack
Patch-mainline: never, kabi
References: kabi

In networking-stable-18_07_27, commit
27cde44a259c380a3c09066fc4b42de7dde9b1ad (tcp: do not cancel delay-AcK
on DCTCP special ACK) removed export of tcp_send_ack.

Reexport the function in case somebody else uses that.

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 net/ipv4/tcp_output.c |    1 +
 1 file changed, 1 insertion(+)

--- a/net/ipv4/tcp_output.c
+++ b/net/ipv4/tcp_output.c
@@ -3590,6 +3590,7 @@ void tcp_send_ack(struct sock *sk)
 {
 	__tcp_send_ack(sk, tcp_sk(sk)->rcv_nxt);
 }
+EXPORT_SYMBOL_GPL(tcp_send_ack);
 
 /* This routine sends a packet with an out of date sequence
  * number. It assumes the other end will try to ack it.
